winml/api/windows.ai.machinelearning.native.idl

```diff
@@ -35,10 +35,11 @@ interface ILearningModelDeviceFactoryNative : IUnknown
     HRESULT CreateFromD3D12CommandQueue(ID3D12CommandQueue * value, [out] IUnknown ** result);
 };
 
-[uuid(c71e953f-37b4-4564-8658-d8396866db0d), object, local]
+[uuid(5da37a26-0526-414b-91e4-2a0fa3ddba40), object, local]
 interface ILearningModelSessionOptionsNative : IUnknown
 {
     HRESULT SetIntraOpNumThreadsOverride(UINT32 intraOpNumThreads);
+    HRESULT SetIntraOpThreadSpinning(boolean allowSpinning);
```

Comment:
The come way of doing this is to add a new interface.

ILearningModelSessionOptionsNative1. This way users can QI for the availiblity of this API.

Reply:
ah I see.